frontend/public/co-fetch.js

```diff
@@ -164,7 +164,12 @@ export const coFetchJSON = (url, method = 'GET', options = {}, timeout) => {
 };
 
 export const coFetchText = (url, options = {}, timeout) => {
-  return coFetchCommon(url, 'GET', options, timeout);
+  return coFetchCommon(url, 'GET', options, timeout).then((response) => {
+    if (_.isObject(response)) {
+      return _.isEmpty(response) ? '' : JSON.stringify(response);
+    }
+    return response;
```

Comment:
```suggestion
    return _.toString(response);
```
Let's enforce this a bit more. `null` => `''`, `false` => `'false'`. I think this is a good safety to have.

Reply:
Udpated.